src/sidebar/components/HypothesisApp.tsx

```diff
@@ -49,6 +49,20 @@ function HypothesisApp({
   const route = store.route();
   const isModalRoute = route === 'notebook' || route === 'profile';
 
+  const roundedCornersEnabled = store.isFeatureEnabled('rounded_corners');
+  useLayoutEffect(() => {
+    const html = document.querySelector('html');
+
+    html?.style.setProperty(
```

Comment:
You might as well use `querySelector('html')!` since I can't think of a legitimate scenario in which this selector will not return an element.